Index: b/cmake/Modules/FindLuajit.cmake
===================================================================
--- a/cmake/Modules/FindLuajit.cmake
+++ b/cmake/Modules/FindLuajit.cmake
@@ -37,7 +37,12 @@ find_path(
     luajit-2.1
     include/luajit-2.1
     luajit2.1
-    include/luajit2.1)
+    include/luajit2.1
+    luajit5.1
+    include/luajit-5_1-2.1
+    moonjit-2.2
+    include/moonjit-2.2
+)
 
 find_library(
   LUAJIT_LIB
